docker-compose.yml

```diff
@@ -118,13 +163,21 @@ services:
       - enroll_assets_cache:/enroll/public/assets
       - ../aca_entities:/aca_entities
       - ../event_source:/event_source
+      # - ${PWD}/hotpatches/keycloack/saml_information.rb:/enroll/config/saml_information.rb:ro
     env_file:
-      - ./site/.env.dev
-      - ./site/rabbitmq/rabbitmq.dev.env
-      - env.dev
+      - path: ./site/.env.dev
+        required: true
+      - path: ./site/rabbitmq/rabbitmq.dev.env
+        required: true
+      - path: env.dev
+        required: true
+      - path: sso.dev.env
+        required: true
+      - path: env.local
```

Comment:
I may have missed the idea of `env.dev` initially - if that is gitignored, `env.local` may just be a point of point of potential confusion. Maybe we yank that? (though re-ordering `env.dev` to _after_ `sso.dev.env` would be useful - I use it for changing `LOGOUT_URL` and `LOGIN_URL` so they work in simple running).

Reply:
100% agree, I think we need to re-structure all the env files, initially I wanted to have a "pre-set" but now we have env files that evaluate at different time setting different scopes

However I think we should do it on a different PR, do you mind if we tackle this next ?